Makefile

```diff
@@ -1636,27 +1636,25 @@ ensure-webassets-e:
 	@if [[ "${WEBASSETS_SKIP_BUILD}" -eq 1 ]]; then mkdir -p webassets/teleport && mkdir -p webassets/e/teleport/app && cp web/packages/teleport/index.html webassets/e/teleport/index.html; \
 	else MAKE="$(MAKE)" "$(MAKE_DIR)/build.assets/build-webassets-if-changed.sh" Enterprise webassets/e/e-sha build-ui-e web e/web; fi
 
-# Enables the pnpm package manager if building webassets is not skipped,
-# pnpm is not already enabled and Corepack is available.
+# Enables the pnpm package manager if  pnpm is not already enabled and Corepack
+# is available.
 # We check if pnpm is enabled, as the user may not have permission to
 # enable it, but it may already be available.
 # Enabling it merely installs a shim which then needs to be downloaded before first use.
 # Corepack typically prompts before downloading a package manager. We work around that
 # by issuing a bogus pnpm call with an env var that skips the prompt.
 .PHONY: ensure-js-package-manager
 ensure-js-package-manager:
-ifneq ($(WEBASSETS_SKIP_BUILD),1)
 	@if [ -z "$(CHECK_PNPM)" ]; then \
 		if [ -n "$(CHECK_COREPACK)" ]; then \
```

Comment:
Thoughts on inlining these two env vars? They're used only in one place and are needed only when someone calls `make build/teleport`. If we inline them, then each invocation of make will not have to call `pnpm --version` and `corepack --version` unnecessarily.